Makefile_packaging.mk

```diff
@@ -324,20 +329,20 @@ chrootbuild: $(SRPM) $(CALLING_MAKEFILE)
 name=$$repo:$$branch:$$build_number\n\
 baseurl=$${JENKINS_URL:-https://build.hpdd.intel.com/}job/daos-stack/job/$$repo/job/$$branch/$$build_number/artifact/artifacts/centos7/\n\
 enabled=1\n\
-gpgcheck = False\n" >> /etc/mock/default.cfg;                                        \
+gpgcheck = False\n" >> /etc/mock/default.cfg;                                    \
```

Comment:
Line continuator not lined up with subsequent lines.

Reply:
Will fix on next update.